backend-rust/src/rest_api.rs

```diff
@@ -178,6 +180,9 @@ enum Balance {
     TotalAmount,
     TotalAmountCirculating,
     TotalAmountUnlocked,
+    TotalAmounUnlocked, /* deprecated: please use 'totalAmountUnlocked' going forward over
+                         * 'totalAmounUnlocked' as 'totalAmounUnlocked' is now deprecated and
+                         * will be removed in a future release */
```

Comment:
This comment makes me worry someone might choose to remove it based on time passed, and it should instead be removed when we see no more activity for this in the API.
```suggestion
    TotalAmounUnlocked, /* deprecated: please use 'totalAmountUnlocked' going forward over
                         * 'totalAmounUnlocked' as 'totalAmounUnlocked' is now deprecated and
                         * should be removed when no more activity is registered for this field */
```